### DISCUSSION TITLE
Kia Uvo / Hyundai Bluelink dialog issue

### DISCUSSION THREAD

**Discussion by NOLBwithaD** [Hyundai-Kia-Connect/kia_uvo#321] [General]
Sorry to trouble you, I have added HACS to HA, and your Kia Uvo / Hyundai Bluelink integration, but nothing I do will open the UI.  Am I being completely thick?  What do I need to do to kick it off?  When I click the integration, it simply opens the Readme.md file.

Thank you.

**Comment by cdnninja** [comment]
Once it’s installed go to integrations and add it like a build in integration.

**Comment by NOLBwithaD** [comment]
Brilliant, that’s fixed it.  Thank you. :-)

**Comment by Aerodynamite** [comment]
I'm very new to Home Assistant and was looking for the UI for very long as well. Perhaps this information can be added to the README?
